tasks/oeth/002-update-l2pao-to-aliased-l1pao/VALIDATION.md

```diff
@@ -0,0 +1,76 @@
+
+Links:
+- [Etherscan](https://optimistic.etherscan.io/address/0x4200000000000000000000000000000000000018)
+- [Optimism Github Repository](https://github.com/ethereum-optimism/optimism/blob/bcdf96abe62da2caaacb0d9571518a7b6c872a37/op-service/predeploys/addresses.go#L23)
+
+State Changes:
+- **Key:** `0x0000000000000000000000000000000000000000000000000000000000000000` <br/>
+  **Before:** `0x0000000000000000000000007871d1187a97cbbe40710ac119aa3d412944e4fe` <br/>
+  **After:** `0x0000000000000000000000006b1bae59d09fccbddb6c6cceb07b7279367c4e3b` <br/>
+  **Meaning:** The `_owner` address variable is set to `0x6B1BAE59D09fCcbdDB6C6cceb07B7279367C4E3b`. The correctness of
+   this slot is attested to in the Optimism repo at [storageLayout/ProxyAdmin.json](https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts-bedrock/snapshots/storageLayout/ProxyAdmin.json#L4). The current owner of the `ProxyAdmin` contract is `0x7871d1187A97cbbE40710aC119AA3d412944e4Fe`, which is a 1-of-1 Safe deployed to closely mirror the production environment configuration. The new `_owner` is `0x6B1BAE59D09fCcbdDB6C6cceb07B7279367C4E3b` which is the aliased L1 Proxy Admin Owner address (unaliased address [`0x5a0Aae59D09fccBdDb6C6CcEB07B7279367C3d2A`](https://etherscan.io/address/0x5a0Aae59D09fccBdDb6C6CcEB07B7279367C3d2A)) i.e. 
+   ```
+   0x5a0Aae59D09fccBdDb6C6CcEB07B7279367C3d2A + 0x1111000000000000000000000000000000001111 = 0x6B1BAE59D09fCcbdDB6C6cceb07B7279367C4E3b
+   ```
+   You can use [chisel](https://book.getfoundry.sh/chisel/) to verify the aliasing result by invoking the `applyL1ToL2Alias` function.
+   ```bash
+   applyL1ToL2Alias(0x5a0Aae59D09fccBdDb6C6CcEB07B7279367C3d2A)
+   ```
+   You can find a reference to this code here: [`applyL1ToL2Alias`](https://github.com/ethereum-optimism/optimism/blob/op-contracts/v1.3.0/packages/contracts-bedrock/src/vendor/AddressAliasHelper.sol#L28).
+
+### `0x7871d1187a97cbbe40710ac119aa3d412944e4fe` (Former L2 `ProxyAdmin` Owner (Safe))
+
+Links:
+- [Etherscan](https://optimistic.etherscan.io/address/0x7871d1187a97cbbe40710ac119aa3d412944e4fe)
+- [Optimism Github Repository](https://github.com/ethereum-optimism/optimism/blob/bcdf96abe62da2caaacb0d9571518a7b6c872a37/op-service/predeploys/addresses.go#L23)
+
+State Changes:
+- **Key:** `0x0000000000000000000000000000000000000000000000000000000000000005` <br/>
+  **Before:** `0x0000000000000000000000000000000000000000000000000000000000000002` <br/>
+  **After:** `0x0000000000000000000000000000000000000000000000000000000000000003` <br/>
+  **Meaning:** The Safe nonce is updated.
+
+
+### `0x420000000000000000000000000000000000001a` (L1FeeVault)
+
+The balance on the `L1FeeVault` increases.
+
+
+### `0x3041ba32f451f5850c147805f5521ac206421623` (Signer on the L2 Proxy Admin Owner Safe)
+
+State Changes:
+- **Nonce** <br/>
+  **Before:** 0 <br/>
+  **After:** 1 <br/>
+  **Meaning:** Account sending the transaction and is the only signer on the safe.
```

Comment:
This line is probably the right spot to account for https://github.com/ethereum-optimism/superchain-ops/pull/213#discussion_r1619395361, since the safe is a 5/7